i18n/qqq.json

```diff
@@ -79,6 +79,16 @@
 	"view-in-browser": "Label for the view in browser link. Shown in the footer",
 	"about-app-message": "Main message in About App page",
 	"softkey-read-more": "Label of the read more softkey",
+	"feedback-header": "Header title shown in Feedback component, selected from Settings menu",
+	"feedback-explanation": "Message displained in Feedback component that explains how feedback information will be used",
```

Comment:
This needs to include explanation of the 2 arguments ($1 and $2)

Check the `confirm-section` msg for an example.